src/frontend/overlay/src/components/organisms/status/Tickers.tsx

```diff
@@ -26,13 +26,14 @@ type JSTickerBodyProps = { pos: number };
 
 const JSTickerBody = styled.div.attrs<JSTickerBodyProps>(({ pos }) => ({
     style: {
-        left: pos,
+        transform: `translate3d(${pos}px, 0, 0)`,
     },
 }))<JSTickerBodyProps>`
     position: absolute;
     width: ${TICKER_WIDTH}px;
     height: 100%;
     background-color: blue;
+    will-change: transform;
```

Comment:
I'm not sure if you need both will-change and translate3d here. Basically, will-change will hint to the browser that you can move this element to it's own gpu layer, which is similar to what translate3d effectively achieves. 

At the same time, will-change will actually add more work to the gpu, and we should be somewhat careful about adding it to a lot of elements. I would just write a simple benchmark (similar to the one above), and check the perfomance difference. Compare the current approach, will-change only, translate3d only and both, and maybe plot it to the amount of vdom elements you are affecting (e.g. n = 100, 1k, 5k, 10k, 50k).